mailbox/store/src/test/java/org/apache/james/mailbox/store/search/AbstractMessageSearchIndexTest.java

```diff
@@ -623,6 +623,36 @@ public void internalDateOnShouldReturnMessagesOfTheGivenDate() throws Exception
     }
 
     @Test
+    public void sentDateAfterShouldReturnMessagesAfterAGivenDate() throws Exception {
+        SearchQuery searchQuery = new SearchQuery();
+        // Date : 2015/06/04 11:00:00.000 ( Paris time zone )
+        searchQuery.andCriteria(SearchQuery.sentDateAfter(new Date(1433408400000L), SearchQuery.DateResolution.Second));
```

Comment:
instead of putting a comment about the date value, please parse it from a readable string